flink-staging/flink-gelly/src/main/java/org/apache/flink/graph/example/ConnectedComponents.java

```diff
@@ -119,23 +113,32 @@ private static boolean parseParameters(String [] args) {
 		return true;
 	}
 
-	@SuppressWarnings("serial")
-	private static DataSet<Edge<Long, NullValue>> getEdgesDataSet(ExecutionEnvironment env) {
-
-		if(fileOutput) {
-			return env.readCsvFile(edgeInputPath)
-					.ignoreComments("#")
-					.fieldDelimiter("\t")
-					.lineDelimiter("\n")
-					.types(Long.class, Long.class)
-					.map(new MapFunction<Tuple2<Long, Long>, Edge<Long, NullValue>>() {
-						@Override
-						public Edge<Long, NullValue> map(Tuple2<Long, Long> value) throws Exception {
-							return new Edge<Long, NullValue>(value.f0, value.f1, NullValue.getInstance());
+	private static Graph<Long, Long, NullValue> getGraph(ExecutionEnvironment env)
+	{
+		Graph<Long, Long, NullValue> graph;
+		if(!fileOutput)
+		{
+			DataSet<Edge<Long, NullValue>> edges = ConnectedComponentsDefaultData.getDefaultEdgeDataSet(env);
```

Comment:
Let's also keep this consistent. In Single Source Shortest Paths you read fromDataSet(getDefault..., env). maybe we could do that for all the examples